packages/console/src/components/CreateTenantModal/SelectTenantPlanModal/index.tsx

```diff
@@ -30,12 +33,17 @@ type Props = {
 function SelectTenantPlanModal({ tenantData, onClose }: Props) {
   const [isSubmitting, setIsSubmitting] = useState<string>();
```

Comment:
Hmmm, this is not new code, but it is a bit unexpected... `isSubmitting` looks like a boolean, but in fact a string.
Can you update it to `submittingPlanId` or something more string-ish?